src/messages/operation_pipe_ambiguous.cr

```diff
@@ -2,11 +2,11 @@ message OperationPipeAmbiguous do
   title "Type Error"
 
   block do
-    text "We cannot determine the order of the operands becuase the pipe makes it ambiguous."
+    text "We cannot determine the order of the operands because the pipe makes it ambiguous."
   end
 
   block do
-    text "You sould add parentheses around the right operands."
+    text "You should add parentheses around the right operand."
```

Comment:
This was intentional, it's not the right (as in side) but as the proper(?) intentional(?) or better how about `You should add parentheses around the right operation.`?

Reply:
Ouch, I wouldn't guess xD By _right_ you mean RHS?

Reply:
Maybe `You should add parentheses around the ambiguous operand.`?

Reply:
Not the RHS, let me give you an example: 
```
10 * 20 |> function() 
```

It can be both operations, I mean the one that is _correct_ in the scope of what it supposed to do. Don't know how to phrase it properly :sweat_smile:

Reply:
I see... 😅 Maybe let's try avoiding the subject, whatcha think about sth like `Wrap operands in parentheses to avoid ambiguity`? ;)

Reply:
> I see... sweat_smile Maybe let's try avoiding the subject, whatcha think about sth like `Wrap operands in parentheses to avoid ambiguity`? ;)

Perfect! :tada: